frontend/sass/_data-driven-onboarding.scss

```diff
@@ -58,7 +58,8 @@
 
 .jf-ddo-results {
     .jf-ddo-card {
-        p.subtitle {
+        p.subtitle, span.subtitle {
```

Comment:
Here, I needed some way to mimic the styling of the paragraphs of text within a DDO card without actually using `<p>` tags. It seems like by default, all blocks of texts in the "Indicators" array get wrapped in a `<p>` tag and then each block of "fallback text" also get's wrapped in one. I wanted to break up the text within both of these and thought it was overkill to refactor things so I added this styling to `<span>` to mimic the spacing of the paragraph. Since `<p>`'s are `display: block` by default, this shouldn't have any effect on pre-existing styling.

Reply:
Ah yeah, that is an annoying thing I ran into earlier too.  This seems like a coolio fix tho and I don't _think_ it should affect accessibility!

Reply:
great!